src/less/donut-graph/donut-graph.less

```diff
@@ -0,0 +1,87 @@
+@import "../variables/variables.less";
+@import "../mixins/private/token-mixins.less";
+@import "../mixins/private/bubble-mixins.less";
+
+@inactive-opacity: 0.2;
+
+.donut-graph {
+    display: block;
+    width: 100%;
+}
+
+.donut-graph svg {
+    display: inline-block;
+    position: relative;
+    vertical-align: middle;
+    width: 100%;
+}
+
+.donut-graph__segment {
+    opacity: 1;
+    transition: opacity 0.3s;
```

Comment:
In this case, there may not be much of a performance boost, but optionally, you could add `will-change: opacity;` here to initialize browser optimization ahead of time for a potential transition.